php/Terminus/Models/Site.php

```diff
@@ -332,6 +333,19 @@ public function fetch(array $options = array()) {
   }
 
   /**
+   * Re-fetches site attributes from the API
+   *
+   * @return void
+   */
+  public function fetchAttributes() {
+    $response = $this->request->simpleRequest(
+      sprintf('sites/%s/settings', $this->get('id'))
+    );
+    $this->attributes = $response['data'];
+    $this->collection->sites_cache->update((array)$response['data']);
```

Comment:
Not the whole object, just the attributes property. (Terminus has always done this AFAIK.) Whenever we use Site#get, we're accessing that cache.

If we don't update it at this point, the user will get an outdated PHP version number.